### DISCUSSION TITLE
Asking a stupid question here, where is the code for merger rate?

### DISCUSSION THREAD

**Discussion by Jacksavior** [TeamCOMPAS/COMPAS#967] [Q&A] [answered]
Dear Professors and Doctors in COMPAS,
I am new to this area and have been trying to find the merger rate in COMPAS's cpp and python code, but I have not been successful. I would greatly appreciate any suggestions or guidance you could provide.
Thank you for your time and assistance.

**Comment by reinhold-willcox** [comment, marked as answer]
Hi @Jacksavior, 

Very sorry about the delay in responding to you! You are most likely looking for the [FastCosmicIntegration module](https://compas.readthedocs.io/en/latest/notebooks/CosmicIntegration.html#6.-Fast-Cosmic-Integration), which combines the merger rate as a function of metallicity with a (user-defined) metallicity-specific star formation rate. 

If you are simply looking for a merger rate estimate from the output of a compas run at fixed metallicity, you can do so with an estimate on the star formation rate (for some given region of interest), and by rescaling the number of detected mergers in compas by the fraction of the IMF that you sampled your run from. Let me know if this is unclear.

Best,
Reinhold
